library/java/io/envoyproxy/envoymobile/engine/types/EnvoyEventTracker.java

```diff
@@ -0,0 +1,7 @@
+package io.envoyproxy.envoymobile.engine.types;
+
+import java.util.Map;
+
+public interface EnvoyEventTracker {
```

Comment:
We also need a new proguard rule for this to keep its methods from getting stripped.